packages/relay/src/lib/eth.ts

```diff
@@ -175,6 +175,10 @@ export class EthImpl implements Eth {
   private readonly estimateGasThrows = process.env.ESTIMATE_GAS_THROWS
     ? process.env.ESTIMATE_GAS_THROWS === 'true'
     : true;
+  private readonly estimateGasIgnoredErrors = [
+    MirrorNodeClientError.messages.INVALID_HEX,
+    MirrorNodeClientError.messages.INVALID_FROM,
+  ];
```

Comment:
Should we move this to the inner scope? _i.e._, inside `estimateGas`?

Reply:
Hmmm, good point. 

It is a matter of trading heap space for execution time of the `estimateGas` method since it will have to initialise this array every time.

Reply:
In this case, I believe both heap space and time would be negligible. I was suggesting this because (at least to me) it's more clear when you have both definition and usage together. Otherwise, at first glance it may look like `estimateGasIgnoredErrors` may be used somewhere else, when it's not the case.

Nevertheless, just a suggestion.

Reply:
I agree with you, readability is prioritized when the difference in complexity is negligible.

I will move the declaration inside the method.